### PR TITLE
Added on-screen keyboard theme settings

### PR DESCRIPTION
**Pull request by kugiigi**
This is dependent on the the pull request in the [keyboard-component](https://github.com/ubports/keyboard-component/pull/57)

### CODE REVIEW

File: plugins/language/ThemeValues.qml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+ * WITHOUT ANY WARRANTY; without even the implied warranties of
+ * MERCHANTABILITY, SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR
+ * PURPOSE.  See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// THIS FILE IS LOADED DIRECTLY BY NAME FROM THE SECURITY-PRIVACY PANEL.
+// IF YOU RENAME IT YOU MUST UPDATE THE REFERENCE THERE.
```
Comment (UniversalSuperBox) [inline plugins/language/ThemeValues.qml]: Hmm, I don't think this file is loaded by the security-privacy panel. You'll need to update the copyright notification to assign copyright of this file to yourself, too.

**Review by UniversalSuperBox — CHANGES_REQUESTED** [review]

File: plugins/language/ThemeValues.qml (hunk 1 above)
Comment (kugiigi) [reply plugins/language/ThemeValues.qml]: I've updated the copyright notification. Kindly review again, not sure if I did them right 😄 
Thanks!

**Review by mariogrip — APPROVED** [review]
LGTM :+1:

**Review by UniversalSuperBox — APPROVED** [review]
See other approval at https://github.com/ubports/keyboard-component/pull/57. Merge both this and that at the same time.

**Pull request merged**